FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -234,6 +233,11 @@ - (NSString *)_fileNameForSelector:(SEL)selector
     fileName = [fileName stringByAppendingFormat:@"_%@", identifier];
   }
 
+  if (_trimTestFromImageName) {
+    NSRange replaceRange = [fileName rangeOfString:@"test"];
```

Comment:
Will this guarantee to only match the first instance, from the start of the string?